packages/daemon/src/context/ContextMutations.js

```diff
@@ -144,6 +140,22 @@ export default class ContextMutations {
     this._context.next({ type: 'peer_changed', peer })
   }
 
+  setContactFeed(userID: ID | string, contactID: ID | string, feed: bzzHash) {
+    const contact = this._context.openVault.identities.getContact(
+      userID,
+      contactID,
+    )
+    if (!contact) throw new Error('User not found')
+
+    contact.contactFeed = feed
```

Comment:
Shouldn't the vault get saved after updating?

Reply:
ditto